oauth/migrations/Version20220211114040.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20220211114040 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add "email" and "password" fields to user table';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE user ADD email VARCHAR(100) NOT NULL, ADD password VARCHAR(100) NOT NULL');
+        $this->addSql('CREATE UNIQUE INDEX unique_user_email ON user (email)');
```

Comment:
I would recomment to follow this naming convention for indexes and foreign keys:
- Indexes: IDX + table name + column(s) => `IDX_table_column`
- Foreign keys: FK + table name + column => `FK_table_column`
- Unique: U + table name + column(s) => `U_table_column`

But just a suggestion to keep consistency across the project :)